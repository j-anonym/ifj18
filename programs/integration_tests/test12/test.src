x=0
while(x != 5) do
y=0
print x
x = x+1
end

print "\n"
x=0
while(x<5.23) do
a=5
print x
x = x+1
end

x=0
print "\n"
while x !=5 do
print x
x = x+1
end
print "\n"
