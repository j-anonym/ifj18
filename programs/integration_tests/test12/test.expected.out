01234
012345
01234
